src/Components/ReadingListBook/ReadingListBook.js

```diff
@@ -4,13 +4,17 @@ import { removeFavorite } from '../../actions'
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux';
 import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
 
 const ReadingListBook = ({ title, image, id, removeFavorite }) => {
 
   return (
     <article className='reading-list-book-card'>
-      <img className="card-image" alt={title} src={image} />
+      <Link to={`/${id}`}>
```

Comment:
Great fix!! I like this solution, so that we don't add more components than we have to.